modules/openshift-developer-cli-sample-applications.adoc

```diff
@@ -0,0 +1,127 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/openshift_developer_cli/creating-a-single-component-application-with-odo.adoc
+
+[id='odo-sample-applications']
+= Sample applications
```

Comment:
Just a thought, but shouldn't we actually move this whole section into the "Understanding odo" page, what are your thoughts? The sample applications as well as examples?

Reply:
What are your thoughts @Preeticp @boczkowska

Reply:
@cdrage for a user who has just started using odo, he would first need to understand what odo does, install it, and then create an application. It would not work if user starts of with `$ odo create java --git https://github.com/spring-projects/spring-petclinic.git` 
Thus, I believe the current placement is right. 
In case the user prefers examples other than Node.js he can directly jump to the other examples using the TOC right at the top of the page.
![Screenshot from 2019-10-11 12-29-50](https://user-images.githubusercontent.com/16647625/66631097-275e1b00-ec23-11e9-9111-5f0de9c571ae.png)